### DISCUSSION TITLE
Apple privacy manifest with uno

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno#16257] [General]
I was wondering if there was any special consideration regarding the Apple Privacy Manifest that we should keep in mind when developing Uno apps, aside from the [general guidance from .NET](https://devblogs.microsoft.com/dotnet/apple-privacy-manifest-support/)? I didn’t see any open discussions on this topic, so I thought I would ask.

**Comment by marwalsch** [comment]
I have done nothing more than following the instructions from the blog post and it worked for me.

**Reply by jeromelaban** [reply]
We've updated our docs to include specific instruction for Uno, but they're indeed very similar to the blog post and docs from Microsoft, it should be published in our public docs very soon here: https://aka.platform/uno/apple-privacy-manifest.
